temperature/models.py

```diff
@@ -0,0 +1,14 @@
+from sqlalchemy import Column, Integer, ForeignKey, DateTime, Float
+from sqlalchemy.orm import relationship
+
+from datebase import Base
```

Comment:
There is a typo in the import statement: 'from datebase import Base'. It should be 'from database import Base'.